README.md

```diff
@@ -259,6 +259,29 @@ GitHub actions will then pick up the branch name and use it to clone that branch
 
 When it's time to merge the sibling PRs, you'll need to merge the prisma/prisma PR first, so when merging the engines PR you have the code of the adapters ready in prisma/prisma `main` branch.
 
+### Testing engines in `prisma/prisma`
+
+You can trigger releases from this repository to npm that can be used for testing the engines in `prisma/prisma` either automatically or manually:
+
+#### Automated integration releases from this repository to npm
+
+(Since July 2022). Any branch name starting with `integration/` will, first, run the full test suite in Buildkite `[Test] Prisma Engines` and, second, if passing, run the publish pipeline (build and upload engines to S3 & R2)
+
+The journey through the pipeline is the same as a commit on the `main` branch.
+- It will trigger [`prisma/engines-wrapper`](https://github.com/prisma/engines-wrapper) and publish a new [`@prisma/engines-version`](https://www.npmjs.com/package/@prisma/engines-version) npm package but on the `integration` tag.
+- Which triggers [`prisma/prisma`](https://github.com/prisma/prisma) to create a `chore(Automated Integration PR): [...]` PR with a branch name also starting with `integration/`
+- Since in `prisma/prisma` we also trigger the publish pipeline when a branch name starts with `integration/`, this will publish all `prisma/prisma` monorepo packages to npm on the `integration` tag.
+- Our [ecosystem-tests](https://github.com/prisma/ecosystem-tests/) tests will automatically pick up this new version and run tests, results will show in [GitHub Actions](https://github.com/prisma/ecosystem-tests/actions?query=branch%3Aintegration)
+
+This end to end will take minimum ~1h20 to complete, but is completely automated :robot:
+
+Notes:
+- in `prisma/prisma` repository, we do not run tests for `integration/` branches, it is much faster and also means that there is no risk of tests failing (e.g. flaky tests, snapshots) that would stop the publishing process.
+- in `prisma/prisma-engines` tests must first pass, before publishing starts. So better keep an eye on them and restart them as needed.
```

Comment:
```suggestion
- in `prisma/prisma-engines` the Buildkite test pipeline must first pass, then the engines will be built and uploaded to our storage in the Buildkite release pipeline. These 2 pipeline can fail for different reason, it's recommended to keep an eye on them (check notifications in Slack) and restart jobs as needed. Finally it will trigger [`prisma/engines-wrapper`](https://github.com/prisma/engines-wrapper). 
```